deploy/cpp_infer/src/args.cpp

```diff
@@ -16,6 +16,7 @@
 
 // common args
 DEFINE_bool(use_gpu, false, "Infering with GPU or CPU.");
+DEFINE_bool(use_mlu, false, "Infering with MLU or CPU.");
```

Comment:
Please include more informative message here, as MLU is not a well known abbreviation yet.